tests/unit.suite.yml

```diff
@@ -4,3 +4,4 @@ bootstrap: false
 modules:
   enabled:
     - Asserts
+    - Yii2
```

Comment:
we dont need Yii2 module to test validator. just give it what it need. inject it as proper dependency